Script/boot/ks4.ks

```diff
@@ -159,8 +160,9 @@ delegates:ADD({
 	debug("delta v: " + delta_v).
 	ADD NODE(time_to_node, 0, 0, delta_v).
 	debug("insertion node: " + NEXTNODE).
-	execute_node(30).
-
+	LOCAL engstat IS engine_breakdown().
+	debug("engstat: " + engstat).
+	execute_grade_node(engstat, 120).
```

Comment:
Not entirely certain this margin parameter is doing what I want it to do, but it seems to need to be larger.